### DISCUSSION TITLE
squash commits?

### DISCUSSION THREAD

**Discussion by Thoragh** [ACINQ/eclair#1997] [Developers]
Hi,

Should I squash fixup commits in Pr or leave them as separate commits?
See #1986 with 2 commits with message "to squash".

**Comment by pm47** [comment]
Hi! No, we squash at merge, see: https://github.com/ACINQ/eclair/blob/master/CONTRIBUTING.md#pull-request-philosophy

> When addressing pull request comments, we recommend using fixup commits. The reason for this is two fold: it makes it easier for the reviewer to see what changes have been made between versions (since Github doesn't easily show prior versions) and it makes it easier on the PR author as they can set it to auto-squash the fixup commits on rebase.
